cob_teleop/ros/src/cob_teleop.cpp

```diff
@@ -382,20 +395,32 @@ void CobTeleop::joy_cb(const sensor_msgs::Joy::ConstPtr &joy_msg){
 
   if(init_button_>=0 && init_button_<(int)joy_msg->buttons.size() && joy_msg->buttons[init_button_]==1)
   {
-    ROS_INFO("Init button pressed");
+    ROS_INFO("Init and recover");
+    say("init and recover", true);
 
     for(std::map<std::string,XmlRpc::XmlRpcValue>::iterator p=components_.begin();p!=components_.end();++p)
     {
       std::string comp_name = p->first;
+      say(comp_name, true);
+
+      ROS_INFO("Init %s",comp_name.c_str());
```

Comment:
addressed in https://github.com/ipa-fmw/cob_command_tools/commit/23cc62115a2c9df18d07c9454f833aff178f5d01